momepy/elements.py

```diff
@@ -170,6 +173,22 @@ def __init__(
         use_dask=True,
         n_chunks=None,
     ):
+        if os.getenv("ALLOW_LEGACY_MOMEPY", "False").lower() not in (
```

Comment:
Although this message varies across differing classes, perhaps one message with formatted bits stored somewhere might be a good idea? Something like:

```
(
    "Class based API like `momepy.{}` is deprecated. "
    "Replace it with `momepy.{}` or "
    "`momepy.{}` to use functional API instead "
    "or pin momepy version <1.0. Class-based API will be removed in "
    "1.0. "
).format(....)
```

Reply:
I'm in the fence now that it's only used in 3 places. Probably OK the way it is unless there are more places we'll need to warn.

Reply:
All three are unique, there's no point in making a template.